packages/core/src/RPA/core/windows/context.py

```diff
@@ -0,0 +1,116 @@
+import contextlib
+import functools
+from typing import Optional
+import logging
+
+from .helpers import IS_WINDOWS
+
+if IS_WINDOWS:
+    from comtypes import COMError  # noqa
+    import uiautomation as auto
+else:
+    COMError = Exception
```

Comment:
Is this really required if the uiautomation module won't be there anyways? or is this a linter thing?

Reply:
More like a linter thing, can be stripped off.
And as a nitpick, `import uiautomation as auto` can go before `from comtypes import COMError  # noqa` for a proper import sorting.